deploy/charts/openebs-monitoring/README.md

```diff
@@ -125,7 +125,7 @@ helm install openebs-monitoring openebs-monitoring/monitoring --namespace openeb
 | `node-problem-detector.metrics.serviceMonitor.enabled`                                    | Enables node problem detector monitoring                                            | `true`                                                    |
 | `alertRules.cStor.enabled`                                                                | Enables openebs pool and volume rules for cStor                                     | `true`                                                    |
 | `alertRules.Jiva.enabled`                                                                 | Enables openebs volume rules for Jiva                                               | `true`                                                    |
```

Comment:
Should be jiva

Reply:
Will take all this configuration changes in the next PR.